packages/react-search-ui-views/src/__tests__/SingleSelectFacet.test.js

```diff
@@ -48,79 +32,31 @@ it("renders", () => {
   expect(wrapper).toMatchSnapshot();
 });
 
-describe("determining selected option from values", () => {
-  it("will correctly determine which of the options is selected based on the provided value", () => {
+describe("determining selected option", () => {
+  it("will correctly determine which of the options is selected", () => {
     const wrapper = render(<SingleSelectFacet {...params} />);
-    expect(wrapper.find(".sui-select__single-value").text()).toEqual("Range 1");
+    expect(wrapper.find(".sui-select__single-value").text()).toEqual("Range 2");
   });
 
-  it("will correctly determine which of the options is selected even if the provided value has differently ordered props", () => {
+  // This shouldn't ever happen, but if it does, it should use the first selected value
+  it("will used the first selected option when multiple options are selected", () => {
```

Comment:
That's true. 

It is possible and valid for two filter values to be selected for a particular facet, because those filters can be set independently of facets ... like via the low level API, or query string parameter. 

The SingleSelectFacet is just a view of that facet and associated selected values. The view will only ever let you select one value, but again, it's possible to select multiple values through other means.

I guess throwing an error instead of using the first selected value would be better behavior because it would help users catch errors, which would otherwise go undetected.

I'm on board, will create an issue for v2 since it would be breaking.

Reply:
https://github.com/elastic/search-ui/issues/438